tests/jeos/image_info.pm

```diff
@@ -50,79 +72,136 @@ sub run {
     }
 
     record_info('HDD', $hdd);
-    ls;
 
     if ($hdd =~ /\.xz/) {
         # We want to monitor the size of uncompressed images.
         my $cmd = "nice ionice unxz -k $hdd -c > hdd_uncompressed";
         record_info('unxz', "Extracting compressed file to get its size.\n$cmd");
         system($cmd);
-        $size = -s 'hdd_uncompressed';
-        ls;    # debug
+        $image_size = -s 'hdd_uncompressed';
         system("rm hdd_uncompressed");
         ($image = $hdd) =~ s/\.xz//;
     } else {
         $image = $hdd;
-        $size = -s $hdd;
+        $image_size = -s $hdd;
     }
-    my $size_mb = $size / 1024 / 1024;
+    my $size_mb = $image_size / 1024 / 1024;
     record_info('Image', "Image: $image\nSize: $size_mb");
 
-    # Record this value in the InfluxDB
+    # Get list of packages installed in the system
+    my $packages = script_output('rpm -qa --queryformat "%{SIZE} %{NAME}\n" |sort -n -r');
+    my @rpm_array = split(/\n/, $packages);
+    my $num_packages = scalar @rpm_array;
+    record_info('rpm total', "Total number of installed packages: $num_packages");
+    record_info('rpm list', $packages);
+
+    # Get the size of different file types
+    my $btrfs_summary = script_output('btrfs filesystem df --mbytes --si /');
+    record_info('btrfs', "$btrfs_summary");
+    my ($data) = $btrfs_summary =~ /Data.*/g;
+    my ($data_mb) = $data =~ /(\d+\.\d+)/g;
+    my ($system) = $btrfs_summary =~ /System.*/g;
+    my ($system_mb) = $system =~ /(\d+\.\d+)/g;
+    my ($metadata) = $btrfs_summary =~ /Metadata.*/g;
+    my ($metadata_mb) = $metadata =~ /(\d+\.\d+)/g;
+    my ($globalreserve) = $btrfs_summary =~ /GlobalReserve.*/g;
+    my ($globalreserve_mb) = $globalreserve =~ /(\d+\.\d+)/g;
+    record_info('Data', "$data_mb MB");
+    record_info('System', "$system_mb MB");
+    record_info('Metadata', "$metadata_mb MB");
+    record_info('GlobalReserve', "$globalreserve_mb MB");
+
+    # Get size of the root directories in KB (except /proc).
+    my $dirs = script_output('du -d 1 --block-size=1K --exclude=/proc /');
```

Comment:
IMO not that useful, except maybe `/usr` and `/var`

Reply:
Yeah, but as I'm running that in some loop way for all dirs, we can always chose to represent which data to display in the Graphs. Otherwise I would need to create a list of directories to analyze which may vary in the future. This way we just collect data for all directories under `/` and later one we show what we want.

Reply:
You already exclude `/proc` and you'd have to exclude some others anyway as the sizes are nonsensical (`/sys`, `/dev/`, `/.snapshots`)...

Reply:
ok, will get the info of those 2 dirs then, thanks for the feedback!

Reply:
Actually, I also forgot about usr-merge only being in TW... For SLE we'd need `/(s)bin` and `/lib(64)` as well. I also forgot about `/boot`... So excluding "special" mounts is probably the best idea.

Reply:
ok, moving back to my original idea :D